apm-sniffer/apm-agent-core/src/test/java/org/apache/skywalking/apm/agent/core/logging/core/PatternLoggerTest.java

```diff
@@ -1,59 +1,67 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
+ * Copyright 2011 the original author or authors.
```

Comment:
The license header is still not right. @kezhenxu94 Look like our class header check is unavailable to test? Becuase CI passed in this definitely wrong case.

Reply:
> The license header is still not right. @kezhenxu94 Look like our class header check is unavailable to test? Becuase CI passed in this definitely wrong case.

I noticed that this morning. It is the `apache-rat` plugin that is responsible to check the license header, but it didn't complain about anything, and I searched around to find that this form of license header is valid in some projects (that are not ASF TLP).

I'll try to make the check strictly, by using the checkstyle header, which is obviously not working now

Reply:
rat is just checking the Apache acceptable header, so this one is good.

But as Apache source codes, I think we need our header style check to make it strictly. Thanks for taking care of this. Expect your PR.